scripts/karma.js

```diff
@@ -28,20 +28,57 @@ function getTestPaths(path) {
   return [path];
 }
 
-const files = getTestPaths(isDocker ? dockerIntegrationTests : integrationTests);
+function filterTests(paths) {
```

Comment:
This is a quick way to enable distributing test run across machines. I was planning to experiment this, but wanted to get some input on this.